scripts/build.js

```diff
@@ -2,7 +2,8 @@ require('shelljs/global');
 
 const package = require('../package.json');
 const prodDependencies = Object.keys(package.jspm.dependencies);
-const devDependencies = Object.keys(package.jspm.devDependencies);
+const devDependencies = Object.keys(package.jspm.devDependencies)
+  .filter((package) => package !== "systemjs-hot-reloader");
```

Comment:
this was a workaround for an issue i found with bundling systemjs-hot-reloader.
https://github.com/alexisvincent/systemjs-hot-reloader/issues/138

I'll see if i can get a reply on what needs to be configured.